test/sql/storage/catalog/test_storage_sequences.cpp

```diff
@@ -19,8 +19,12 @@ TEST_CASE("Use sequences over different runs without checkpointing", "[storage]"
 		REQUIRE_NO_FAIL(con.Query("CREATE SEQUENCE seq_cycle INCREMENT 1 MAXVALUE 3 START 2 CYCLE;"));
 		result = con.Query("SELECT nextval('seq')");
 		REQUIRE(CHECK_COLUMN(result, 0, {1}));
+		result = con.Query("SELECT currval('seq')");
+		REQUIRE(CHECK_COLUMN(result, 0, {1}));
 		result = con.Query("SELECT nextval('seq_cycle')");
 		REQUIRE(CHECK_COLUMN(result, 0, {2}));
+		result = con.Query("SELECT currval('seq')");
+		REQUIRE(CHECK_COLUMN(result, 0, {2}));
```

Comment:
Correct me if I'm wrong but this test seems incorrect; why is the result of currval('seq') 2 here? It should still be 1, no?

Reply:
Has line 24 not set the value to 2?

Reply:
That uses a different sequence (nextval('seq_cycle')), no?

Reply:
oops, will fix